README.md

```diff
@@ -39,6 +39,11 @@ Our [contribution guide](https://github.com/wildfly-security/wildfly-elytron/blo
  
 For more details, check out our [getting started guide](https://wildfly-security.github.io/wildfly-elytron/getting-started-for-developers/) for developers.
 
+Example Feature demos
+---------------------
+
+Our [elytron-examples](https://github.com/wildfly-security-incubator/elytron-examples) repo contains example demos of WildFly Elytron features.
```

Comment:
Small nitpick here too! Could we substitute ```repo -> repository```?

Reply:
updated, thanks!